src/main/resources/entity-types/inventory/simple_holdings_records.json5

```diff
@@ -642,7 +642,7 @@
       name: 'number_of_items',
       sourceAlias: 'hrd',
       dataType: {
-        dataType: 'stringType',
+        dataType: 'numberType',
```

Comment:
Do you need a cast in the value getter? IIRC the json has a string

Reply:
@ncovercash yup done